cabot/celery_queue_config.py

```diff
@@ -0,0 +1,47 @@
+from collections import defaultdict
+
+from cabot.cabotapp import models
+from cabot.metrics import models
+
+
+NORMAL_CHECK_QUEUE = 'normal_checks'
+CRITICAL_CHECK_QUEUE = 'critical_checks'
+DEFAULT_CHECK_QUEUE = NORMAL_CHECK_QUEUE
+
+STATUS_CHECK_TO_QUEUE = \
+    defaultdict(lambda: defaultdict(lambda: DEFAULT_CHECK_QUEUE), {
+        models.GraphiteStatusCheck.check_category :
+            defaultdict(lambda: DEFAULT_CHECK_QUEUE, {
+                models.CheckGroupMixin.WARNING_STATUS  : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.ERROR_STATUS    : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.CRITICAL_STATUS : CRITICAL_CHECK_QUEUE,
+            }),
+        models.HttpStatusCheck.check_category :
+            defaultdict(lambda: DEFAULT_CHECK_QUEUE, {
+                models.CheckGroupMixin.WARNING_STATUS  : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.ERROR_STATUS    : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.CRITICAL_STATUS : CRITICAL_CHECK_QUEUE,
+            }),
+        models.JenkinsStatusCheck.check_category :
+            defaultdict(lambda: DEFAULT_CHECK_QUEUE, {
+                models.CheckGroupMixin.WARNING_STATUS  : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.ERROR_STATUS    : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.CRITICAL_STATUS : CRITICAL_CHECK_QUEUE,
+            }),
+        models.TCPStatusCheck.check_category :
+            defaultdict(lambda: DEFAULT_CHECK_QUEUE, {
+                models.CheckGroupMixin.WARNING_STATUS  : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.ERROR_STATUS    : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.CRITICAL_STATUS : CRITICAL_CHECK_QUEUE,
+            }),
+    })
+
+METRIC_CHECK_TO_QUEUE = \
+    defaultdict(lambda: defaultdict(lambda: DEFAULT_CHECK_QUEUE), {
+        models.ElasticsearchStatusCheck.check_category :
+            defaultdict(lambda: DEFAULT_CHECK_QUEUE, {
+                models.CheckGroupMixin.WARNING_STATUS  : NORMAL_CHECK_QUEUE,
+                models.CheckGroupMixin.ERROR_STATUS    : CRITICAL_CHECK_QUEUE,
+                models.CheckGroupMixin.CRITICAL_STATUS : CRITICAL_CHECK_QUEUE,
+            }),
+    })
```

Comment:
Can we just combine STATUS_CHECK_TO_QUEUE and METRIC_CHECK_TO_QUEUE? I don't really see a reason to have separate dicts